### ISSUE TITLE
"Gevent is not installed" error always on MacOS 10.14.6

### ISSUE THREAD

**Issue by greatvc** [morpheus65535/bazarr#515]
Hi, just discover this and i'm amazed! using Sonarr and Radarr on my MacMini since 2015 with no issues.

Tried to install Bazarr, did everything, all are ok installed but on start i get 

```python
python bazarr.py
Bazarr starting...
ERROR:root:BAZARR require gevent Python module to be installed using pip.
Traceback (most recent call last):
  File "bazarr/main.py", line 31, in <module>
    import gevent
ImportError: No module named gevent
Bazarr exited.
```

As you can see both gevent and greenlet is installed

```python
sudo pip --no-cache-dir install gevent
Password:
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Requirement already satisfied: gevent in /Users/<>/Library/Python/2.7/lib/python/site-packages (1.4.0)
Requirement already satisfied: greenlet>=0.4.14; platform_python_implementation == "CPython" in /Users/<>/Library/Python/2.7/lib/python/site-packages (from gevent) (0.4.15)
```

```python
Requirement already satisfied: greenlet in /Users/<>/Library/Python/2.7/lib/python/site-packages (0.4.15)
```

```python
10:bazarr <>$ pip --version
pip 19.2.2 from /Library/Python/2.7/site-packages/pip-19.2.2-py2.7.egg/pip (python 2.7)
```
Any help will be greatly appreciated.

Regards,
Vassilis

**Comment by morpheus65535** [comment]
What is the output of `python --version`?

**Comment by greatvc** [comment]
Here you go Morpheus. I today downloaded/installed everything for bazarr for the firsttime

```
python --version
Python 2.7.10
```

**Comment by morpheus65535** [comment]
https://github.com/morpheus65535/bazarr/wiki/Installation-Running-from-Source

> bazarr require Python 2.7.13 or greater (version 3.x not supported) and can be run from source.

**Comment by greatvc** [comment]
ok i seriously don't know how i installed that cause i copy paste the links from your wiki :nerd_face: 

Now i get this error:

```python
python bazarr.py
Bazarr starting...
Traceback (most recent call last):
  File "bazarr/main.py", line 75, in <module>
    update_notifier()
  File "/Applications/bazarr/bazarr/notifier.py", line 13, in update_notifier
    a = apprise.Apprise()
  File "/Applications/bazarr/bazarr/../libs/apprise/Apprise.py", line 78, in __init__
    self.locale = AppriseLocale()
  File "/Applications/bazarr/bazarr/../libs/apprise/AppriseLocale.py", line 111, in __init__
    self.lang = AppriseLocale.detect_language(language)
  File "/Applications/bazarr/bazarr/../libs/apprise/AppriseLocale.py", line 208, in detect_language
    lang = locale.getdefaultlocale()[0]
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/locale.py", line 545, in getdefaultlocale
    return _parse_localename(localename)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/locale.py", line 477, in _parse_localename
    raise ValueError, 'unknown locale: %s' % localename
ValueError: unknown locale: UTF-8
```

**Comment by morpheus65535** [comment]
I would try this result from Google:
https://coderwall.com/p/-k_93g/mac-os-x-valueerror-unknown-locale-utf-8-in-python

**Comment by greatvc** [comment]
YEAP!
you also need to reload the settings
```python
source ~/.bash_profile
```
And voila  bazarr is working.

Ok very very fast question, i run the commands from a terminal window, which i see it needs to remain open, how to make it you know, not run from there or not have it open all the time ?  :cool:

**Comment by morpheus65535** [comment]
I don't know how to do it on MacOS, sorry.

**Comment by greatvc** [comment]
No problem, i'm closing the issue!
Many thanks for your great support !!!

**Comment by caronc** [comment]
@morpheus65535 ; just with respect to this error:
```
python bazarr.py
Bazarr starting...
Traceback (most recent call last):
  File "bazarr/main.py", line 75, in <module>
    update_notifier()
  File "/Applications/bazarr/bazarr/notifier.py", line 13, in update_notifier
    a = apprise.Apprise()
  File "/Applications/bazarr/bazarr/../libs/apprise/Apprise.py", line 78, in __init__
    self.locale = AppriseLocale()
  File "/Applications/bazarr/bazarr/../libs/apprise/AppriseLocale.py", line 111, in __init__
    self.lang = AppriseLocale.detect_language(language)
  File "/Applications/bazarr/bazarr/../libs/apprise/AppriseLocale.py", line 208, in detect_language
    lang = locale.getdefaultlocale()[0]
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/locale.py", line 545, in getdefaultlocale
    return _parse_localename(localename)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/locale.py", line 477, in _parse_localename
    raise ValueError, 'unknown locale: %s' % localename
ValueError: unknown locale: UTF-8
```

This issue is also on me and ties back to #478; the good news it's been fixed in Apprise v0.7.9. :slightly_smiling_face:

**Comment by morpheus65535** [comment]
@caronc Thank you, I'll include 0.7.9 in next release of Bazarr.
